es-app/src/guis/GuiMenu.h

```diff
@@ -16,8 +16,16 @@ class GuiMenu : public GuiComponent
 
 private:
 	void addEntry(const char* name, unsigned int color, bool add_arrow, const std::function<void()>& func);
-	void openScreensaverOptions();
+	void addVersionInfo();
 	void openCollectionSystemSettings();
+	void openConfigInput();
```

Comment:
Thank you for this very much needed split. GuiMenu was becoming quite unmanageable.